### PR TITLE
Feature #11 main navigation

### LINKED ISSUE #11: Main Navigation
# REQUIREMENTS
Design and implement project main navigation between screeens

# DESIGN
Check for all the project screens

# Acceptance Criteria
List all required acceptance criteria
- [ ] The navigation include all the requested screens
- [ ] The navigation works correctly between every single screen

### PR DESCRIPTION
**Pull request by vmagana3**
# Description
we implemented the main navigation project without autentication just to see navigation between screens
 
 # Testing
There´s nothing to test
 
  
  ## Screenshots 
screencast-Genymotion-2021-01-18_19 05 07 085

### CODE REVIEW

File: flightreservation/components/AgreeItems.js [hunk 1]
```diff
@@ -70,12 +70,14 @@ const styles = StyleSheet.create({
   containerSpacing: {
     marginTop: 30,
   },
-  buttonDivider: {
+  /*
```
Comment (JacintoOrtiz15745) [inline flightreservation/components/AgreeItems.js:73]: Remember don't submit code with comments

**Review by Joaquin-Niembro — APPROVED** [review]

**Comment by Joaquin-Niembro** [comment]
nice video preview of the app!

**Review by jjcr11 — APPROVED** [review]
Good, i like the example with video form :+1:

**Review by JacintoOrtiz15745 — APPROVED** [review]
It looks good, well done!

**Pull request merged**